astronomer/providers/apache/hive/sensors/hive_partition.py

```diff
@@ -20,7 +20,10 @@ class HivePartitionSensorAsync(HivePartitionSensor):
 
        Since we use `implya <https://github.com/cloudera/impyla>`_ library,
        please set the connection to use the port ``10000`` instead of ``9083``.
-       This sensor currently supports ``auth_mechansim='PLAIN'`` only.
+       For ``auth_mechansim='GSSAPI'`` the ticket renewal happens through command
+       ``airflow kerberos`` in
+       `worker/trigger <https://airflow.apache.org/docs/apache-airflow/stable/security/kerberos.html>`_.
+       Host map entry in worker/trigger for Public address to Private dns address of the hive cluster.
```

Comment:
Can you elaborate on this Host map entry please?

Reply:
I have tried to improve it here 75310921adda9589fa53d685a25b5ca8ddb02f08. Please let me know how it looks.